bigtable/google/cloud/bigtable/table.py

```diff
@@ -257,6 +257,22 @@ def list_column_families(self):
             result[column_family_id] = column_family
         return result
 
+    def get_cluster_states(self):
+        """List the cluster states owned by this table.
+
+        :rtype: dict
+        :returns: Dictionary of cluster states for this table.
+        Keys are cluster ids and values are
+        :class: '_ClusterState' instances
+        """
```

Comment:
Please align lines 265 and 266 above by the letter `D` on line 264 to make docs test happy.